api/contrib/envoy/extensions/filters/http/golang/v3alpha/golang.proto

```diff
@@ -142,44 +29,47 @@ message Config {
     OVERRIDE = 3;
   }
 
-  // library_id is a unique ID for a dynamic library file, must be unique globally.
+  // Globally unique ID for a dynamic library file.
   string library_id = 1 [(validate.rules).string = {min_len: 1}];
 
-  // Dynamic library implementing the interface of
-  // ``StreamFilter <contrib/golang/filters/http/source/go/pkg/api.StreamFilter>``.
+  // Path to a dynamic library implementing the
+  // :repo:`StreamFilter API <contrib/golang/filters/http/source/go/pkg/api.StreamFilter>`
+  // interface.
   // [#comment:TODO(wangfakang): Support for downloading libraries from remote repositories.]
   string library_path = 2 [(validate.rules).string = {min_len: 1}];
 
-  // plugin_name is the name of the go plugin, which needs to be consistent with the name
-  // registered in http::RegisterHttpFilterConfigFactory.
+  // Name of the Go plugin.
```

Comment:
does this have to be unique in any way ?

Reply:
Yes.

Reply:
unique to what ? globally ?

Reply:
> unique to what ? globally ?

Globally unique.